src/ol/format/filter/Bbox.js

```diff
@@ -30,7 +30,11 @@ class Bbox extends Filter {
     /**
      * @type {import("../../extent.js").Extent}
      */
-    this.extent = extent;
```

Comment:
So that the type annotation applies to this property, you should leave this line here.

You can throw in a condition below (without needing an `else`):

```js
if (extent.length !== 4) {
    throw new Error('Expected an extent with four values ([minX, minY, maxX, maxY])';
}
```